actions/actions.py

```diff
@@ -113,10 +113,7 @@ def slot_mappings(self) -> Dict[Text, Union[Dict, List[Dict]]]:
                 self.from_entity(entity="job_function"),
                 self.from_text(intent="enter_data"),
             ],
-            "use_case": [
-                self.from_text(intent="enter_data"),
-                self.from_intent(intent="ask_faq_voice", value="voice"),
-            ],
+            "use_case": [self.from_text(intent="enter_data"),],
```

Comment:
are we sure about this? I think people can say "a voice bot" in response to the use case

Reply:
ohhh yes true. damn it

Reply:
retrieval actions are kind a double edged sword